src/Annotation/VoteUpDownWidget.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\vud\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Vote Up/Down Widget annotation object.
+ *
+ * @see \Drupal\vud\Plugin\VoteUpDownFieldManager
+ * @see plugin_api
+ *
+ * @Annotation
+ */
+class VoteUpDownWidget extends Plugin {
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $plugin_id;
+
+  /**
+   * The label of the plugin.
```

Comment:
I am not getting the description part quite a bit.
Okay so let's take an example:
Plain pluign:
admin_plugin = 'plain'
description = 'Plain widget type'??? 
Is it alright? If it is, then I guess the description doesn't make much sense